cybershell/scoring.py

```
@@ -0,0 +1,482 @@
+        if variance < min_improvement:
+            return True
+
+        # Pivot if trend is declining
+        if self.get_trend() == 'declining':
+            return True
+
+        # Pivot if we've been stuck below a threshold
+        if self.get_sma() < 0.3 and len(self.scores) > patience * 2:
+            return True
+
+        return False
```

Ruff (0.12.2)

Return the condition directly

Inline condition

(SIM103)